python/paddle/distributed/fleet/fleet.py

```diff
@@ -362,6 +363,245 @@ def init(
                 )
         return self
 
+    # test allreduce perf
+    def allreduce_test(
```

Comment:
use `allreduce_perf` or `allreduce_check` will be better？same issue as `reduce_scatter_test` etc below

Reply:
> use `allreduce_perf` or `allreduce_check` will be better？same issue as `reduce_scatter_test` etc below

has rename xxx_test to xx_perf